### ISSUE TITLE
documentations for driver class

### ISSUE THREAD

**Issue by jeongukjae** [tensorflow/tfx#6695]
## URL(s) with the issue:

* https://github.com/tensorflow/tfx/blob/master/tfx/dsl/components/base/base_driver.py
* https://www.tensorflow.org/tfx/guide/custom_component

### Clear description:

When customizing TFX components, sometimes Driver class is very important. But couldn't find docs for the Driver class for detailed use-case. So it may helpful for advanced users.

**Comment by briron** [comment]
Yeah, I think we missed the documentation of the Driver class. Thanks for pointing this out.

Before documentation, I want to clarify if you need a general explanation of the Driver class or there is a specific example that you want to know from this. Any clarification will help us to make better docs.

**Comment by jeongukjae** [comment]
@briron thanks. I could understand by following course and tfx GitHub.

* https://www.cloudskillsboost.google/course_templates/191

The concept was clear, but it should be great if the docs clarify compatibilities (supported runners). I dig into the Driver for my custom component, and found that it is not supported in pipelines that running in Vertex AI (https://github.com/tensorflow/tfx/issues/6722).

**Comment by jeelania10** [comment]
Hi @jeongukjae, Thank you so much for showing interest! We truly appreciate your enthusiasm and support.
We regret to inform you that we are not accepting new implementation requests or feature additions at this time. Please feel free to fork the repository and submit pull requests if you have any suggestions for enhancements, problem fixes, or ideas. We’d be more than happy to review and collaborate! If you would like to contribute , please follow our [CONTRIBUTION GUIDELINES](https://github.com/tensorflow/tfx/blob/master/CONTRIBUTING.md) here and submit a pull request.
Thank you again for your interest, and we look forward to seeing your contributions!

**Comment by jeelania10** [comment]
Closing this due to inactivity. Please feel free to reopen and post your comments (if you still have queries on this). Thank you!
